src/axom/slic/interface/slic.cpp

```diff
@@ -18,11 +18,23 @@ namespace slic
 //------------------------------------------------------------------------------
 bool debug::checksAreErrors = false;
 
-void initialize() { Logger::initialize(); }
+static bool root = true;
```

Comment:
... or alternatively

```suggestion
static bool s_is_root = true;
```